core/src/main/java/org/mule/el/mvel/MVELExpressionLanguage.java

```diff
@@ -69,7 +71,11 @@ public void initialise() throws InitialisationException
     {
         parserConfiguration = createParserConfiguration();
         expressionExecutor = new MVELExpressionExecutor(parserConfiguration);
+    }
 
+    @Override
+    public void start() throws MuleException
+    {
         loadGlobalFunctions();
```

Comment:
it was assumed that the global functions and their dependencies were already initialised which was not necessarily the case with the new lifecycle. However, now that the "initialisation tree" has been implemented, I can revisit this change and see if this is still necessary.